views/login.html

```diff
@@ -7,7 +7,7 @@
 {% block content %}
 <div class="vertical-center">
   <div class="container">
-    <h1>Sign up or Login &ndash; It's Free</h1>
+    <h1>Sign up or Login and start publishing! &ndash; It's Free</h1>
```

Comment:
I would say "Start Publishing - It's Free!" which is the main message here. WDYT @zelima ?